pkg/namesgenerator/names-generator_test.go

```diff
@@ -1,12 +1,14 @@
-package namesgenerator // import "github.com/docker/docker/pkg/namesgenerator"
+package namesgenerator_test // import "github.com/docker/docker/pkg/namesgenerator"
```

Comment:
Not sure we need blackbox testing for this; this probably also needs the `// import` comment to be updated accordingly.

Reply:
I did it too.